monsteer/plugin/spikeReport.cpp

```diff
@@ -49,19 +50,26 @@ namespace
 const uint32_t INTERNAL_TIMEOUT = 500;
 }
 
+URI toHostAndPort( const URI& uri )
+{
+    URI out;
+    out.setHost( uri.getHost( ));
+    out.setPort( uri.getPort( ));
+    return out;
+}
+
 SpikeReport::SpikeReport( const brion::SpikeReportInitData& pluginData )
-    : _uri( pluginData.getURI( ))
+    : _uri( toHostAndPort( pluginData.getURI( )))
```

Comment:
Agh!, I didn't noticed that.

I think the simplest solution is to add a zeq::URI constructor that takes a host and port.